organization/infrastructure.md

```diff
@@ -7,19 +7,37 @@ Research for which internal tools to use should be made as [RFC/decision style d
 - file sharing (tresorit, syncthing?)
 - bug tracking (linear)
 - source code hosting (github)
-- CI (?)
+- CI (_may become practical in radicle_ )
 - sysadmin/devops (pyinfra) (see also network topologies (tailscale etc.) in hacking.org > netsec)
 
 ```{tip}
 investigate [Radicle](https://radicle.xyz/blog/towards-decentralized-code-collaboration.html) for source code hosting and bug-tracking
 - <https://radicle.xyz/2024/09/10/radicle-1.0.html>
 - <https://radicle.xyz/guides/user>
+
+Radicle started working on CI, there is a very status update on it
+More on https://radicle.zulipchat.com/#narrow/channel/369274-general/topic/Radicle.20CI.20demo
+```
+
+```{note}
+🙈 , I would like to understand what prevented us to set this up ...
+So far we have tried to set up matrix client element (mac) and elementX (iOS) for the following room
+https://matrix.to/#/!fmBiADJQowvGELBzzb:matrix.org?via=matrix.org
+
+I have been able to connect with my account but wackou did not receive an invitation.
+
+Let's try to make this work - either with element/elementX or using other clients
 ```
 
 ## Security
 
 Security needs to be enforced all the time, we want to have proper access to resources. Nothing "shared all" or public by default. Take care of access with (potentially old) SSH keys on servers.
 
+```{note}
+Needs development. Not sure I understand the specific context here. 
+
+Going down the rabbit hole, for indivuals in collective, common set of authentication tools? (2FA, Yubikey, passkey, pass/bitwarden, ssh/gpg key settings, key rotation practices).
+```
```

Comment:
yeah, the previous paragraph was quickly thrown together without much thought, but the general idea I wanted to convey is that we should be conscious about security and access to our infrastructure, and use modern and proper access to it (eg: we can't have a shared password even if convenient, we should go for something like SSO with 2FA, etc.)

Basically, security is important and it shouldn't take the back seat because it's not convenient to do things properly.